### PR TITLE
nginx: build with http_slice_module

### PR DESCRIPTION
**Pull request by AluminumTank**
Add support for slicing. Runtime tested on my server, and builds for armv7l as well.

### CODE REVIEW

File: srcpkgs/nginx/template [hunk 1]
```diff
@@ -2,7 +2,7 @@
 pkgname=nginx
 reverts="1.17.7_1"
```
Comment (ericonr) [inline srcpkgs/nginx/template:3]: Could you remove this reverts line as well, please?

**Comment by Duncaen** [comment]
You added a build option, but the option is not used.

**Review by Piraty — COMMENTED** [review]
nginx builds without `ngx_http_slice_module` [by default](http://nginx.org/en/docs/http/ngx_http_slice_module.html). should Void enable it by default?

**Comment by AluminumTank** [comment]
I don't think that just because the documentation says it doesn't build by default, we shouldn't have it in our build.

A good example of this is the [SSL module](http://nginx.org/en/docs/http/ngx_http_ssl_module.html), which is not included in the default build. Should Void not include SSL either since it's not default? 

For reference, one of my nginx server blocks uses slice, which is why I'm here. Anecdotally, the build instructions for the package providing this server block config didn't even mention that nginx needs to be built with slice support, so the status quo is that this is a "default" module.

There is likely no easy way to get slice (or other module) support without manually recompiling. The only way I saw was to use dynamic modules, but this requires grabbing the source code for the version of nginx currently on your machine, compiling the modules as dynamic objects, and adding them into the nginx config. This would cause breakage or additional work when the nginx version that Void distributes gets bumped.

Other distributions include this module in their builds as well (I just looked at the PKGBUILD for Arch and ebuild for Gentoo). 

The other option is to package the modules separately as dynamic modules as mentioned above, which I'd be happy to do if this is a better solution.

**Comment by ericonr** [comment]
@AluminumTank that would be a big reworking of the nginx package, so perhaps better left to a separate PR.

@the-maldridge ok to merge?

**Pull request merged**